node/task.ts

```diff
@@ -1775,6 +1803,218 @@ export class CodeCoverageEnabler {
     }
 }
 
+//-----------------------------------------------------
+// Logging Commands
+//-----------------------------------------------------
+
+/**
+ * Upload user interested file as additional log information 
+ * to the current timeline record.
+ * 
+ * The file shall be available for download along with task logs.
+ * 
+ * @param path      Path to the file that should be uploaded.
+ * @returns         void
+ */
+export function uploadFile(path: string) {
+    command("task.uploadfile", null, path);
```

Comment:
Should have ```assertAgent("1.101")```

Reply:
The problem is that you can't assert the agent version lower than 2.104.1